articles/azure-monitor/app/tutorial-asp-net-core.md

```diff
@@ -0,0 +1,525 @@
+
+## Run the Azure Cafe web application
+
+After the web application code is deployed, telemetry will flow to Application Insights. The Application Insights SDK automatically collects incoming web requests to your application.
+
+1. Right-click the **AzureCafe** project in Solution Explorer and select **Publish** from the context menu.
+
+    ![The Visual Studio Solution Explorer displays with the Azure Cafe project selected and the Publish context menu item highlighted.](./media/asp-net-core/web_project_publish_context_menu.png "Publish Web App")
+
+2. Select **Publish** to promote the new code to the Azure App Service.
+
+    ![The AzureCafe publish profile displays with the Publish button highlighted.](./media/asp-net-core/publish_profile.png "Publish profile")
+
+3. Once the publish has succeeded, a new browser window opens to the Azure Cafe web application.
+
+    ![The Azure Cafe web application displays.](./media/asp-net-core/azure_cafe_index.png "Azure Cafe web application")
+
+4. Perform various activities in the web application to generate some telemetry.
+
+   1. Select **Details** next to a Cafe to view its menu and reviews.
+
+        ![A portion of the Azure Cafe list displays with the Details button highlighted.](./media/asp-net-core/cafe_details_button.png "Azure Cafe Details")
+
+   2. On the Cafe screen, select the **Reviews** tab to view and add reviews. Select the **Add review** button to add a review.
+
+        ![The Cafe details screen displays with the Add review button highlighted.](./media/asp-net-core/cafe_add_review_button.png "Add review")
+
+   3. On the Create a review dialog, enter a name, rating, comments, and upload a photo for the review. Once completed, select **Add review**.
+
+        ![The Create a review dialog displays.](./media/asp-net-core/create_a_review_dialog.png "Create a review")
+
+   4. Repeat adding reviews as desired to generate additional telemetry.
+
+### Application map
+
+The sample application makes calls to multiple Azure resources, including Azure SQL, Azure Blob Storage, and the Azure Language Service (for review sentiment analysis).
+
+![The Azure Cafe sample application architecture displays.](./media/asp-net-core/azure-cafe-app-insights.png "Azure Cafe sample application architecture")
+
+Application Insights introspects incoming telemetry data and is able to generate a visual map of detected system integrations.
+
+1. Access and log into the [Azure Portal](https://portal.azure.com).
+
+2. Open the sample application resource group `application-insights-azure-cafe`.
+
+3. From the list of resources, select the `azure-cafe-insights-{SUFFIX}` Application Insights resource.
+
+4. Select **Application map** from the left menu, beneath the **Investigate** heading. Observe the generated Application map.
+
+    ![The Application Insights application map displays.](./media/asp-net-core/application-map.png "Application map")
+
+### Live metrics
```

Comment:
I'd put Live Metrics even before Application Map, as this is the fastest way to verify things are working!

Reply:
Fixed.